tests/classes/Utils.js

```diff
@@ -135,7 +135,7 @@ describe('Utils', () => {
 
       expect(() => {
         serverless.utils.readFileSync(tmpFilePath);
-      }).to.throw(new RegExp(`in "${tmpFilePath}"`));
+      }).to.throw(new RegExp('YAMLException:'));
```

Comment:
how about we simply check for `invalid.yml` in the message? Then the compare works easily across OS and the only really important part is the filename anyway. Its also unlikely to get to a point where the filename is shown, but the path isn't (and even than thats not a big issue because it still clearly shows which file).